libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
Is this commit supposed to be in this PR ? I dont see it, does it compile under IAR? I assume you either remove the header from compilation fro IAR or otherwise we end up with an error..